docs/source/API/core/view/view.rst

```diff
@@ -1,65 +1,64 @@
 ``View``
-========
+============
 
 .. role:: cppkokkos(code)
     :language: cppkokkos
 
 Header File: ``<Kokkos_Core.hpp>``
 
-Usage
------
+Description
+--------------
+
+The programming model's main data structure is the Kokkos View.  The View has multi-dimensional (up to eight) array-like semantics and operations.  It, and related data structures (*e.g.*, `View-like Types <view_like.html>`__)  can be used on the CPU (host) and/or GPU (device), and has architecture-specific optimiztions, shared-memory parallelism, compile-time layouts and memory space.   
```

Comment:
This is mostly not quite right. View does not have array-like semantics or operations actually. It also doesn't have shared-memory parallelism per-se, and I wouldn't necessarily call it the programming model's main data structure - which has some unfortunate connotations.